Makefile

```diff
@@ -181,23 +181,37 @@ tests/%.uplc.run: tests/%.uplc
 	$(KPLUTUS) run $< > $<.out
 	$(CHECK) $<.out $<.expected
 
+# Compare K output against expected output generated from the uplc tool
+# Do this by printing the results of the K cell only.
+tests/%.uplc.runConformance: tests/%.uplc
+	$(KPLUTUS) run $< --result-only > $<.out
+	$(CHECK) $<.out $<.expected
+
+
```

Comment:
We don't need a separate `runConformance` target I don't think. Just add the `--result-only` argument to the `.run` version, we don't have need for the non-result only variant at the moment.

Reply:
we actually use the non-result-only option, it was a part of #63 and the expected result is logged in this file: https://github.com/runtimeverification/plutus-core-semantics/blob/master/tests/simple/prog.uplc.expected